### PR TITLE
added new filters to api query, parse files in seed

### PR DESCRIPTION
**Pull request by sofofos**
A bit of refactoring to optimize the seed file, and parsing the files to map out just the food name and food ids

### CODE REVIEW

File: db/seed_helper.rb [hunk 1]
```diff
@@ -1,16 +1,14 @@
 require 'open-uri'
 require 'json'
-# require 'dotenv'
-# Dotenv.load('.env')
+require 'dotenv'
```
Comment (chrononaut76) [inline db/seed_helper.rb]: Please comment out lines 3 and 5. I found out that these don't run in production.

File: db/seed_helper.rb [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 
-url = "https://api.edamam.com/api/food-database/v2/parser?app_id=#{ENV.fetch('EDAMAM_APP_ID')}&app_key=#{ENV.fetch('EDAMAM_API_KEY')}&nutrition-type=cooking"
-
-5.times do |index|
+Dotenv.load('.env') # run the script from the repo root: ruby db/seed_helper.rb
+filters= "&diet=balanced&dishType=Bread&dishType=Cereals&dishType=Main%20course&dishType=Preps&dishType=Salad&dishType=Sandwiches&dishType=Side%20dish&dishType=Soup&dishType=Starter&imageSize=THUMBNAIL"
+url = "https://api.edamam.com/api/recipes/v2?type=public&app_id=#{ENV.fetch('EDAMAM_RECIPE_API_APP_ID')}&app_key=#{ENV.fetch('EDAMAM_RECIPE_API_KEY')}#{filters}"
+4.times do |index|
```
Comment (chrononaut76) [inline db/seed_helper.rb]: Why the change from `5.times` to `4.times`?

File: db/seed_helper.rb [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-# require 'dotenv'
-# Dotenv.load('.env')
+require 'dotenv'
 
-url = "https://api.edamam.com/api/food-database/v2/parser?app_id=#{ENV.fetch('EDAMAM_APP_ID')}&app_key=#{ENV.fetch('EDAMAM_API_KEY')}&nutrition-type=cooking"
-
-5.times do |index|
+Dotenv.load('.env') # run the script from the repo root: ruby db/seed_helper.rb
+filters= "&diet=balanced&dishType=Bread&dishType=Cereals&dishType=Main%20course&dishType=Preps&dishType=Salad&dishType=Sandwiches&dishType=Side%20dish&dishType=Soup&dishType=Starter&imageSize=THUMBNAIL"
+url = "https://api.edamam.com/api/recipes/v2?type=public&app_id=#{ENV.fetch('EDAMAM_RECIPE_API_APP_ID')}&app_key=#{ENV.fetch('EDAMAM_RECIPE_API_KEY')}#{filters}"
```
Comment (chrononaut76) [inline db/seed_helper.rb:9]: Are `EDAMAM_RECIPE_API_APP_ID` and `EDAMAM_RECIPE_API_KEY` different than the ones already in Heroku?

**Review by chrononaut76 — CHANGES_REQUESTED** [review]
Please see comments inline.

File: db/seed_helper.rb (hunk 3 above)
Comment (sofofos) [reply db/seed_helper.rb:9]: Yes, they are keys for the [Recipe API](https://developer.edamam.com/edamam-recipe-api), which is different from the[ Food Database API ](https://developer.edamam.com/food-database-api)

File: db/seed_helper.rb (hunk 2 above)
Comment (sofofos) [reply db/seed_helper.rb]: To avoid overloading the DB since we are creating these items 8 times (for each store)- we could reduce the number of stores to increase the number of food items, but I can also revert it back to `5.times` if you prefer

Comment (sofofos) [reply db/seed_helper.rb]: increased the num of calls in [d4b09bf](https://github.com/chrononaut76/rails-shopwise/pull/108/commits/d4b09bf61e65e35c644106ea84f92e3104de54e7)

File: db/seed_helper.rb (hunk 1 above)
Comment (sofofos) [reply db/seed_helper.rb]: fixed in [1236af3](https://github.com/chrononaut76/rails-shopwise/pull/108/commits/1236af3a7c9e3f399bb83c64ab00e8b6747532f6)

File: db/seed_helper.rb (hunk 3 above)
Comment (chrononaut76) [reply db/seed_helper.rb:9]: Thanks for clarifying. I'll see about updating the keys on Heroku.

**Review by chrononaut76 — APPROVED** [review]

File: db/seed_helper.rb (hunk 3 above)
Comment (sofofos) [reply db/seed_helper.rb:9]: @chrononaut76 the keys have been added to the config var settings on heroku ✅
